diagnostic.js

```diff
@@ -30,25 +30,68 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+'use strict';
+
+const models = require('app/models');
+const Movement = models.movement;
+
+const create = (name, description, startYear, endYear) => {
+  Movement.create(name, description, startYear, endYear)
+    .then(movement => {
+      return movement.save();
```

Comment:
`.create` already combines `.new` and `.save` - it's not necessary to `.save` again.